include/boost/sml/aux_/utility.hpp

```diff
@@ -309,26 +309,26 @@ struct zero_wrapper<R (*)(TArgs...), T> {
 };
 
 #if defined(__cpp_noexcept_function_type)  // __pph__
-template <class R, class TBase, class... TArgs>
-struct zero_wrapper<R (TBase::*)(TArgs...) noexcept> {
+template <class R, class TBase, class... TArgs, class T>
+struct zero_wrapper<R (TBase::*)(TArgs...) noexcept, T> {
   explicit zero_wrapper(R (TBase::*ptr)(TArgs...) noexcept) : ptr{ptr} {}
   auto operator()(TBase &self, TArgs... args) { return (self.*ptr)(args...); }
 
  private:
   R (TBase::*ptr)(TArgs...) noexcept {};
 };
 
-template <class R, class TBase, class... TArgs>
-struct zero_wrapper<R (TBase::*)(TArgs...) const noexcept> {
+template <class R, class TBase, class... TArgs, class T>
```

Comment:
I think we should introduce tests for `wrap` to ensure the quality of the feature